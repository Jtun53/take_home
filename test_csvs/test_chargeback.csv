type, client, tx, amount
deposit, 1, 1, 1.0
deposit, 2, 2, 2.0
deposit, 1, 3, 2.0
deposit, 2, 4, 5.0
dispute, 2, 2
resolve, 2, 2
dispute, 1, 3
chargeback, 1, 3