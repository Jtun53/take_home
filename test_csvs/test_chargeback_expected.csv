Client, Available, Held, Total, Locked
1, 1.0, 0, 1.0, True
2, 7.0, 0.0, 7.0, False